pkg/codegen/internal/test/sdk_driver.go

```diff
@@ -145,31 +157,72 @@ func TestSDKCodegen(t *testing.T, language string, genPackage GenPkgSignature, c
 				schemaPath = filepath.Join(dirPath, "schema.yaml")
 			}
 
-			files, err := GeneratePackageFilesFromSchema(schemaPath, genPackage)
+			files, err := GeneratePackageFilesFromSchema(schemaPath, opts.GenPackage)
 			require.NoError(t, err)
 
-			// Check output is valid code (will type-check). If code will not
```

Comment:
I'm actually actively preferring it the other way. PULUMI_ACCEPT=1 should let the system materialize the codegen outputs so we can see them in git diff. If subsequent checks fail that's good too.